### ISSUE TITLE
evaluate if we can specify different precompiled laxar entry points in package.json

### ISSUE THREAD

**Issue by jpommerening** [LaxarJS/laxar#425]
@x1B, you once mentioned https://github.com/defunctzombie/package-browser-field-spec in chat.

I think we should check if webpack supports the object syntax, so that we can specify precompiled entry points for `polyfills`, `laxar-compatibility` and `laxar-widget-service-mocks`.
If that works, we should also revisit the webpack externals in `laxar-mocks`. Currently `laxar-mocks` includes its own copy of `laxar/laxar-widget-service-mocks` because it is not "externalized".

If webpack offers full support for the "browser field spec" we could add the following the `package.json`:

```json
"browser": {
    "./laxar.js": "./dist/laxar.js",
    "./laxar-compatibility.js": "./dist/laxar-compatibility.with-deps.js",
    "./laxar-widget-service-mocks.js": "./dist/laxar-widget-service-mocks.js",
    "./polyfills.js": "./dist/polyfills.js"
}
```

**Comment by jpommerening** [comment]
It _seems_ like it's working as expected.

With the snippet above, I can require `laxar/laxar-compatibility` (directly or by aliasing `laxar$` to it) and I get the file from `dist/`. I also tried to update `laxar-mocks` to use an "external" `laxar/laxar-widget-service-mocks` dependency, which seems to work as well.

I'll commit this, hoping for the best.

@x1B, @alex3683: let me know if you encounter any problems.

**Comment by jpommerening** [comment]
Implemented on `master` (`v2.0.0`)

**Comment by jpommerening** [comment]
Hmpf … seems like Travis doesn't like it.
Can anyone reproduce the issues Travis is having? :(
